src/include/distributed/multi_server_executor.h

```diff
@@ -36,6 +36,12 @@
 #define JOB_CLEANUP_QUERY "SELECT task_tracker_cleanup_job("UINT64_FORMAT ")"
 #define JOB_CLEANUP_TASK_ID INT_MAX
 
+/* Adaptive executor repartioning related defines */
+#define WORKER_CREATE_SCHEMA_QUERY "SELECT worker_create_schema \
+ ("UINT64_FORMAT ");"
```

Comment:
the string formatting is a little strange here. Should this be?

```c
"SELECT worker_create_schema" \
"(" UINT64_FORMAT ")";
```

(unless it fits on a single line?)